src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -152,7 +152,14 @@ public ResponseEntity<GetHearingResponse> getHearingRequest(Long hearingId, bool
                 throw new HearingNotFoundException(hearingId, HEARING_ID_NOT_FOUND);
             }
         } else {
-            return ResponseEntity.noContent().header("Content-Length", "0").build();
+            Integer latestRequestVersion = 0;
+            Optional<HearingEntity> hearingEntity = hearingRepository.findById(hearingId);
+            if (hearingEntity.isPresent()) {
+                latestRequestVersion = hearingEntity.get().getLatestRequestVersion();
+            }
+            return ResponseEntity.noContent()
+                    .header("latestHearingRequestVersion", String.valueOf(latestRequestVersion))
+                    .build();
```

Comment:
Please can we not default it to 0. We can use a similar approach to a few lines above where it returns the ResponseEntity or throws HearingNotFoundException. Can make it more robust and easier to read too:
```suggestion
            HearingEntity hearingEntity = hearingRepository.findById(hearingId)
                .orElseThrow(() -> new HearingNotFoundException(hearingId, HEARING_ID_NOT_FOUND));
            return ResponseEntity.noContent()
                .header("Latest-Hearing-Request-Version", 
                        String.valueOf(hearingEntity.getLatestRequestVersion()))
                .build();
```

Reply:
Done